src/Parser.php

```diff
@@ -3430,10 +3427,7 @@ private function parseInterfaceDeclaration($parentNode) {
         return $interfaceDeclaration;
     }
 
-    /**
-     * @return InterfaceMembers
-     */
-    private function parseInterfaceMembers($parentNode) : Node {
+    private function parseInterfaceMembers($parentNode) : InterfaceMembers {
```

Comment:
probably dozens of other places we can do this, but figure PHPStan will pick this up on level 4 or 5